third homework.py

```diff
@@ -0,0 +1,89 @@
+# 1 calc
+# do we need some func for that?
+
+while True:
+    first_number = input("Put your 1st number:")
+    if first_number == "quit":
+        break
+    first_sign = input("Put your arithmetic sign:")
+    if first_sign == "quit":
+        break
+    second_number = input("Put your 2nd number:")
+    if second_number == "quit":
+        break
+    if first_sign == "+":
+        result = first_number + second_number
+    elif first_sign == "-":
+        result = first_number - second_number
+    elif first_sign == "*":
+        result = float(first_number) * float(second_number)
+    elif first_sign == "/":
+        if float(second_number) != 0:
+            result = float(first_number) / float(second_number)
+        else:
+            print("Divided by zero is prohibited")
+            continue
+    else:
+        print("Something went wrong")
+        continue
+
+    print(result)
+
+# 2 list divider
+
+my_list = list(input("Put your list_values here: "))
+
+if len(my_list) % 2 == 0:
+    # why I cant make first_l_s var as INT?
+    first_list_size = len(my_list) // 2
+    second_list_size = len(my_list) // 2
+    print(f"list(First half:, {my_list[:first_list_size]}, Second half: {my_list[first_list_size:]}) ")
+elif len(my_list) % 2 != 0:
+    first_list_size = len(my_list) // 2 + 1
+    second_list_size = len(my_list) // 2
+    print(f"list(First half:, {my_list[:first_list_size]}, Second half: {my_list[first_list_size:]}) ")
+elif len(my_list) == 0:
+    print(f"list(First half:, {my_list}, Second half: {my_list}) ")
+
```

Comment:
good, buy please be precise with your output, the task says that output should be a list which consist of two "halfs of the start list

so answer has to be [my_list[:first_list_size], my_list[first_list_size:]]